manifests/pipelines/aiedge-e2e.pipeline.yaml

```diff
@@ -136,7 +136,22 @@ spec:
     - input: "$(params.fetch-model)"
       operator: in
       values: ["git"]
-  # Use kserve-download-model container image to fetch from S3 storage
+  - name: move-model-to-root-dir
+    params:
+    - name: model-name
+      value: $(params.model-name)
+    - name: subdirectory
```

Comment:
This is confusing as the `git clone` task also has a predefined subdirectory. Maybe we have it as model-directory? Or something else. Just that the subdirectory is where git clone by default clones the repository into.

Reply:
But the subdirectory here and in gitclone mean the same thing

Reply:
For the git-fetch step is what I am saying

Reply:
In that case, you format the `/model_dir-$(params.model-name)/` I know it might be just a formatting change but the consistency would be helpful for others reading this.